react/package.json

```diff
@@ -46,6 +46,7 @@
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-replace": "^2.2.0",
+    "sass-loader": "^8.0.0",
```

Comment:
Whats this change for?

Reply:
i missed a dep in the react restructure, discussed irl